vcd/resource_vcd_vm.go

```diff
@@ -22,25 +25,29 @@ func resourceVcdStandaloneVm() *schema.Resource {
 }
 
 func resourceVcdStandaloneVmCreate(_ context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	startTime := time.Now()
+	util.Logger.Printf("[DEBUG] [VM create] started standalone VM creation")
+	if d.Get("vapp_name").(string) != "" {
+		return diag.Errorf("vApp name must not be set for a standalone VM (resource `vcd_vm`)")
+	}
+
+	util.Logger.Printf("[DEBUG] [VM create] finished standalone VM creation")
```

Comment:
This log would be the same as on line 42, wouldn't it?

Reply:
Yes. This was a leftover - good catch.